scania-digital-design-system/sdds
# fix(dropdown): various improvements

**Describe pull-request**  
Various fixes and improvements for Dropdown component

**Solving issue**  
Fixes:

1. [AB#1675](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1675)
2. [AB#1673](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1673)
3. [AB#1358](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1358), #270 
4. [AB#1386](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1386), #297 
5. #304 
6. #284 
7. Propagate selected-value and selected-text on the top of the parent as attributes.
8. [AB#1669](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1669)

**How to test**  
1. Go to Storybook
2. Check ticket per ticket from "Fixes" if it is still valid
